README.md

```diff
@@ -1,70 +1,52 @@
-# Azure SDK for Javascript
+# Azure SDK for JavaScript
+
 
 | Component            | Build Status                                                                                                                                                                                |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Management Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/138?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=138&branchName=master) |
 | Client Libraries     | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/45?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=45&branchName=master)   |
 
-This project provides an isomorphic Javascript package with TypeScript definitions that makes it easy to consume and manage
-Microsoft Azure Services.
-It supports SDKs for:
-
-- ARM services (control plane) (packages with the naming convention of `@azure/arm-<servicename>`)
-- data plane of some Azure services (packages with the naming convention of `@azure/<servicename>`).
-
-## Documentation
-
-Documentation of the supported SDKs can be found here:
-
-- https://docs.microsoft.com/en-us/javascript/azure - This website primarily provides SDK documentation for
-
-  - ARM based services (`@azure/arm-<serviceName>`)
-  - data plane SDKs like `@azure/batch`, `@azure/graph`, etc.
+This repository contains official JavaScript libraries and TypeScript definitions for Azure services. For documentation go to [Azure SDK for JavaScript documentation](https://aka.ms/js-docs).
 
-## Authentication
+You can find a complete list of all the packages for these libraries [here](https://github.com/Azure/azure-sdk-for-js/blob/master/packages.md).
 
-- For Node.js-based authentication, look at [@azure/ms-rest-nodeauth](https://npmjs.com/package/@azure/ms-rest-nodeauth).
-- For browser-based authentication, look at [@azure/ms-rest-browserauth](https://npmjs.com/package/@azure/ms-rest-browserauth).
-  - The browser authentication storage is a little more complicated, so we encourage you to [read about how it works](https://github.com/Azure/ms-rest-browserauth/blob/master/README.md) before putting it in your application.
+## Getting started
 
-## Need Help?
+For your convenience, each service has a separate set of libraries that you can choose to use instead of one, large Azure package. To get started with a specific library, see the **README.md** file located in the library's project folder. You can find service libraries in the `/sdk` directory.
```

Comment:
This section on its own is very confusing

I'd either move this section under the "Packages available" section or remove the "Packages available" header altogether.

Reply:
One way to simplify:

The `sdk` folder in this repository lists each Azure service for which we have libraries supported. You can choose to use these on their own instead of one, large Azure package. 

Each such service might have multiple packages from each of the following categories

...
...
....

To get started with a specific library, see the **README.md** file located in the library's project folder.

Reply:
I like removing the packages available header. The language tweaks I like but depart from the standard template.  @kaerm any issues with removing the "packages available" header?

Reply:
No problem at all - the template is to make sure all the information is there but it is in no way supposed to be a hard constraint. As long as you explain the types of packages we have (people do not understand data vs management plane distinction), how to navigate the repos, and specifically call out the preview ones I think it should be fine